sql_generators/mobile_kpi_support_metrics/templates/attribution_clients.view.sql

```diff
@@ -0,0 +1,29 @@
+{{ header }}
+CREATE OR REPLACE VIEW
```

Comment:
Should we have submission_date on the view? In particular given it's the partition for the table

Reply:
I don't think it's strictly necessary here and partitioning would still be used correctly when getting the data from the underlying table if someone applied the filter on the view. For now I added it to reduce the confusion.